app/controllers/home_controller.rb

```diff
@@ -18,14 +18,14 @@ def status
 
   # check for exception - set the result code and duration time
   def exception_watch(&block)
-    start = Time.now.utc
+    start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
I would create a method that returns it in simpler form, for example:
```
def monotonic_time
  Process.clock_gettime(Process::CLOCK_MONOTONIC)
end
```

It makes the reading easier because you use it in more then a single location.

Reply:
The problem with one-line helper is it needs to go into util.rb, so it will end up as `Foreman::Util.monotonic_time` which is as long as the original line and it's IMHO actually less readable because you need to look into util.rb and figure out what this is. Plus people will forget about helper method the moment this PR is merged. Calling `clock_gettime` is also educative - it informs readers about how to measure time properly.

I'd definitely create a helper if it was more complex code or we plan on using it at many places, but this looks like all we have in our codebase.